src/components/ServiceDetails/AboutService/DemoToggler.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { withStyles } from "@material-ui/styles";
+
+import StyledButton from "../../common/StyledButton";
+import DemoExample from "./DemoExample";
+import Routes from "../../../utility/constants/Routes";
+import { useStyles } from "./styles";
+
+const DemoToggler = ({ classes, showDemo, onClick, service }) => {
```

Comment:
How about `DemoToggle` instead.

Reply:
Toggler is the common suffix we use for all the conditional rendering components..

Reply:
`Toggle` or a `Switch` is a commonly used name for such components. If it is too much of a task, we could re-visit this later.